lib/stripe_model_callbacks/models/stripe_invoice.rb

```diff
@@ -6,6 +6,8 @@ class StripeInvoice < StripeModelCallbacks::ApplicationRecord
 
   has_many :stripe_invoice_items, autosave: true, primary_key: "stripe_id"
 
+  validates_uniqueness_of :stripe_id
```

Comment:
I recall the new recommended way of doing validations is like this:
```suggestion
  validates :stripe_id, uniqueness: true
```